easybuild/easyconfigs/r/Reapr/Reapr-1.0.18-cleanup.patch

```diff
@@ -0,0 +1,219 @@
+This patch makes Reapr compatible with EasyBuild.
+
+Henrik R. Nagel, 20191115
+diff -ruN Reapr_1.0.18.orig/Makefile Reapr_1.0.18/Makefile
+--- Reapr_1.0.18.orig/Makefile	1970-01-01 01:00:00.000000000 +0100
++++ Reapr_1.0.18/Makefile	2019-11-13 12:17:03.000000000 +0100
+@@ -0,0 +1,8 @@
++all:
++	cd third_party/tabix ; make
++	cd third_party/snpomatic ; make
++	mkdir -p bin/tabix ; cp third_party/tabix/tabix third_party/tabix/bgzip third_party/tabix/tabix++ bin/tabix
++	mkdir bin/snpomatic ; cp third_party/snpomatic/findknownsnps bin/snpomatic
++	cp third_party/smalt_x86_64 bin/smalt
++	cd src ; make
++	cd bin ; ln -s reapr.pl reapr
+diff -ruN Reapr_1.0.18.orig/src/Makefile Reapr_1.0.18/src/Makefile
+--- Reapr_1.0.18.orig/src/Makefile	2014-03-13 11:15:43.000000000 +0100
++++ Reapr_1.0.18/src/Makefile	2019-11-13 12:38:07.000000000 +0100
+@@ -1,6 +1,5 @@
+-BAMTOOLS_ROOT = $(CURDIR)/bamtools
+ CC = g++
+-CFLAGS = -Wl,-rpath,$(BAMTOOLS_ROOT)/lib  -Wall -O3  -I $(BAMTOOLS_ROOT)/include -L $(BAMTOOLS_ROOT)/lib
++CFLAGS = -Wl,-rpath,$(EBROOTBAMTOOLS)/lib  -Wall -O3  -I $(EBROOTBAMTOOLS)/include/bamtools -L $(EBROOTBAMTOOLS)/lib
```

Comment:
If possible, please use the `$CFLAGS` set by EasyBuild, rather than leaving `-O3` hardcoded

Reply:
Done. I have removed all hardcoded CFLAGS and similar and fixed the Makefile rules accordingly. I have pushed new versions of Reapr-1.0.18-foss-2019a.eb and Reapr-1.0.18-cleanup.patch.